pkg/cloud/gcp/actuators/machine/reconciler.go

```diff
@@ -117,30 +114,63 @@ func (r *Reconciler) create() error {
 		Items: metadataItems,
 	}
 
-	operation, err := r.computeService.InstancesInsert(r.projectID, zone, instance)
+	return instance, nil
+}
+
+// Create creates machine if and only if machine exists, handled by cluster-api
```

Comment:
This comment is confusing and I'm not sure what it means.

Reply:
Yep - that was there before though. :-) :-1:

Reply:
Done in https://github.com/openshift/cluster-api-provider-gcp/commit/f3071fbeefc793d7b15ec1d45178e71138081ac7